install/traf_tools_setup.sh

```diff
@@ -43,6 +43,7 @@
 #   common services such as naming, configuration management, synchronization, 
 #   and group services.
 # Thrift: Communications and data serialization tool
+# Maven: Build tool that is only installed if compatible version does not exist
```

Comment:
Actually, that version check is commented out below, maybe change this comment as well?